packages/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-settings-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingSettingsPlugin.kt

```diff
@@ -11,26 +11,30 @@ open class ExpoAutolinkingSettingsPlugin : Plugin<Settings> {
     // Creates an extension that allows users to link expo modules and add additional configuration.
     settings.extensions.create("expoAutolinking", ExpoAutolinkingSettingsExtension::class.java, settings)
 
-    settings.gradle.beforeRootProject { rootProject ->
-      // Adds the `expo-autolinking-plugin` to the root project, so it will be available for all subprojects.
-      rootProject
-        .buildscript
-        .dependencies
-        .add("classpath", "expo.modules:expo-autolinking-plugin")
-    }
-
     val expoGradlePluginPath = getExpoGradlePluginPath(settings)
-    // Includes the `expo-gradle-plugin` subproject.
-    settings.includeBuild(
-      expoGradlePluginPath
-    )
+    // If the `expo-gradle-plugin` is available, it will be included in the settings.
+    // It won't be available in our test project or when we decide to prebuild plugin.
```

Comment:
nit: if user does not setup settings.gradle, is that possible to have `expoGradlePluginPath` non-exist? in such case, how to about show a warning?

Reply:
Currently, it is not possible in a standard scenario because using the plugin requires including the build folder, which needs to be available. However, in our tests, we typically do not include the project as it is provided by the Gradle test SDK.